test/prime generator/prime.chpl

```diff
@@ -0,0 +1,46 @@
+/* Primality Test */
```

Comment:
The Wikipedia entry with the pseudocode for this algorithm seems to think it needs more citations than they currently provide, so I would suggest adding a comment with where you got the algorithm from, just to be nice :)